astropy/coordinates/tests/test_shape_manipulation.py

```diff
@@ -34,6 +34,10 @@ class TestManipulation():
     """
 
     def setup_class(cls):
+        # This is to be consistent with setting in astropy/conftest.py
+        # that somehow is not getting picked up.
+        iers_conf.auto_download = False
```

Comment:
No sure what happened: Either it is not picking up setting from `astropy/conftest.py` or something else is overwriting the setting. But forcing it (again) here seems to work.